src/sinks/prometheus/exporter.rs

```diff
@@ -75,6 +75,8 @@ pub struct PrometheusExporterConfig {
     #[serde(default = "default_flush_period_secs")]
     #[serde_as(as = "serde_with::DurationSeconds<u64>")]
     pub flush_period_secs: Duration,
+    #[serde(default = "default_suppress_timestamp")]
```

Comment:
```suggestion
    #[serde(default)]
```

IIRC the default here is `false`, so we could drop the function as well?